waspc/data/Generator/templates/server/src/auth/providers/email/resetPassword.ts

```diff
@@ -17,15 +17,15 @@ export async function resetPassword(
 
     const { token, password } = args;
     try {
-        const { id: authId } = await verifyToken(token);
+        const { id: providerUserId } = await verifyToken(token);
```

Comment:
But wait, here you know the `providerUserId` is really `email`, right? So why don't you just call it `email`?